stl/inc/type_traits

```diff
@@ -1804,6 +1804,38 @@ inline constexpr bool is_nothrow_invocable_r_v =
     _Select_invoke_traits<_Callable, _Args...>::template _Is_nothrow_invocable_r<_Rx>::value;
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+#ifndef __clang__ // TRANSITION, LLVM-Bug #: To Be Filed
+// STRUCT TEMPLATE is_layout_compatible
+template <class _Ty1, class _Ty2>
+struct is_layout_compatible : bool_constant<__builtin_is_layout_compatible(_Ty1, _Ty2)> {};
+
+template <class _Ty1, class _Ty2>
+inline constexpr bool is_layout_compatible_v = __builtin_is_layout_compatible(_Ty1, _Ty2);
+
+// STRUCT TEMPLATE is_pointer_interconvertible_base_of
+template <class _Base, class _Derived>
+struct is_pointer_interconvertible_base_of
+    : bool_constant<__builtin_is_pointer_interconvertible_base_of(_Base, _Derived)> {};
+
+template <class _Base, class _Derived>
+inline constexpr bool is_pointer_interconvertible_base_of_v = __builtin_is_pointer_interconvertible_base_of(
+    _Base, _Derived);
+
+// FUNCTION TEMPLATE is_pointer_interconvertible_with_class
+template <class _ClassTy, class _MemberTy>
+_NODISCARD constexpr bool is_pointer_interconvertible_with_class(_MemberTy _ClassTy::*_Pm) noexcept {
+    return __builtin_is_pointer_interconvertible_with_class(_ClassTy, _Pm);

```

Comment:
Current state: hundreds of errors from `/BE` compiles complaining that `_ClassTy` on this line is a type, which I suppose suggests that nobody ever taught EDG this intrinsic?

Reply:
Disabled the /BE tests and filed bug on EDG